<launch>
    <node pkg="atro2_vision" type="atro2_cam.py" name="image_srv" output="screen" ></node>
    <node pkg="atro2_vision" type="atro2_line_following.py" name="line_following" output="screen"></node>
</launch>